templates/contrib/paragraphs/timeline/paragraph--timeline-item--roadmap.html.twig

```diff
@@ -39,49 +39,79 @@
  */
 #}
 
-<div class="item">
-    <dd class="item__content">
-        <div class="timeline-slot-header">
-            <h4 class="timeline-slot-title">
-                <button aria-expanded="false"
-                        aria-controls="timeline-slot-content--{{ paragraph.id() }}"
-                        data-controls-img="timeline-slot-img--{{ paragraph.id() }}"
-                        class="accordion--button">{{ content.title }}</button>
-            </h4>
-            <h5 class="timeline-slot-date">{{ content.subtitle }}</h5>
-        </div>
-        <div class="timeline-slot-content accordion--content" id="timeline-slot-content--{{ paragraph.id() }}" aria-hidden="true">
-            <div class="content-image hidden-desktop">
-                {% if content.field_image|render is not empty %}
-                    <div class="image-gallery--wrapper single">
-                        {{ content.field_image }}
-                    </div>
-                {% endif %}
-                {% if content.field_video|render is not empty %}
-                    <div class="video-wrapper">
-                        <div class="video-inner-wrapper">
-                            {{ content.field_video }}
-                        </div>
-                    </div>
-                {% endif %}
-            </div>
-            {{ content.field_text }}
-        </div>
-    </dd>
-    <dd class="item__image-desktop show-desktop-only">
-        <div class="timeline-slot-content timeline-slot-content__img accordion--content" id="timeline-slot-img--{{ paragraph.id() }}" aria-hidden="true">
+{% set classes = [
+    'timeline-slot',
+] %}
+
+{% set note_type = 'important' %}
```

Comment:
Important note zou een ckeditor style worden (of dus meerdere styles). Daarom is hier geen apart veld voor en wordt dit ingegeven in de het wysiwyg tekstveld.

Reply:
Is aangepast in Fractal en ook in deze branch